hostthedocs/templates/index.html

```diff
@@ -52,6 +52,11 @@ <h3>{{ project.name }}</h3>
                     </div>
                 </div>
             </div>
+            {% if loop.index % 4 == 0 %}
```

Comment:
How does this work on medium? Looks like we're using a 2-column grid for medium screens. What happens if we do this every 2nd item rather than 4th? What if we put a `clearfix`class on the row container?

Reply:
So I'll be the first to say I only know enough bootstrap to google and hack around which is how I came up with this fix.  Testing on a 2 column grid seemed to work as expected for a simple test case (I actually had to change the DEFAULT_PROJECT_DESCRIPTION string to an empty string in filekeeper.py to re-create the usecase in #32 where there were no descriptions):
![image](https://user-images.githubusercontent.com/8078693/79295988-c6e4ba80-7e9f-11ea-8c87-51333d5e7b3a.png)

- Putting clearfix every 2nd item makes the grid sit in two column mode even with a full screen browser.
![image](https://user-images.githubusercontent.com/8078693/79296092-0b705600-7ea0-11ea-83d2-ec103f9f290a.png)

- Putting the clearfix on the row container (or each of the columns) has the same problem as the original code.